openacademy/models/openacademy_course.py

```diff
@@ -13,11 +13,11 @@ class Course(models.Model):
     _sql_constraints = [
         ("description_and_title_different",
             "CHECK(name != description)",
-            "The title and description of the course must be different."),
+            _("The title and description of the course must be different.")),
```

Comment:
SQL constraint messages are translatable by default.

Same for the below message

Reply:
Do I remove the `_()` function to the constraints?

Reply:
Yes, but only from SQL constraints. Python constraints do need it